tests/unit/test_session.py

```diff
@@ -3192,7 +3192,9 @@ def test_create_model_from_job_with_vpc_config_override(sagemaker_session):
 
 def test_endpoint_from_production_variants(sagemaker_session):
     ims = sagemaker_session
-    ims.sagemaker_client.describe_endpoint = Mock(return_value={"EndpointStatus": "InService"})
+    ims.sagemaker_client.describe_endpoint = Mock(
+        return_value={"EndpointStatus": "InService", "EndpointArn": "arn:aws:sagemaker:"}
```

Comment:
nit: Could we use a full ARN like `arn:aws:sagemaker:us-west-2:123456789012:endpoint/test`